app/graphql/types/event_type.rb

```diff
@@ -1,13 +1,15 @@
 module Types
   class EventType < Types::BaseObject
 
-    description 'An Event that is run by a Parter'
-
     field :id, ID, null: false
-    field :description, String
+    field :name, String, method: :summary
```

Comment:
Maybe just add a comment here like

# Summary and name are aliases, this is left as a convenience for people used to iCal format